source/assets/public/index.css

```diff
@@ -236,7 +221,6 @@ body {
     transform: none;
   }
   .hero__date {
-    font-family: var(--font-subheading);
```

Comment:
No point loading a font for one thing

Makes it look a little different

Before:
<img width="669" alt="screen shot 2017-09-08 at 6 25 52 pm" src="https://user-images.githubusercontent.com/1615322/30198708-34e16dd6-94c3-11e7-88bd-ea409f4bf01a.png">

After:
<img width="659" alt="screen shot 2017-09-08 at 6 25 44 pm" src="https://user-images.githubusercontent.com/1615322/30198716-3db95658-94c3-11e7-8a9f-e124f431c1eb.png">

Reply:
Weird kiwi artifacts removed in #41